src/bugreporter/odemis_bugreporter.py

```diff
@@ -318,6 +318,17 @@ def compress_files(self):
             if delphi_calib_reps and (time.time() - os.path.getmtime(delphi_calib_reps[-1])) / 3600 < 24:
                 files.append(delphi_calib_reps[-1])
 
+            # Add the latest fastem-calibration image if it's possibly related (ie, less than a week old)
+            fastem_calib_dir = os.path.join(home_dir, "development/fastem-calibrations/images/cell-translation")
```

Comment:
It feels a little odd that the data is stored in `~/development/`. That's a directory that should only be present if developing the software or debugging it. On a normal/stable system, I would assume it should not be there.
Would it be possible to move this directory to `~/fastem-calibration/` (or `~/odemis-status/fastem-calibration/`)? It seems it's just a matter of adjusting the variable in `fastem_calibrations.util` :smile: 

BTW, don't you want to add the whole `~/odemis-status/` directory, while you're at it?